src/d3-cloud.controller.js

```diff
@@ -8,47 +8,43 @@
   d3CloudController.$inject = ['$scope'];
 
   function d3CloudController($scope) {
-    $scope.$watch('words', function(newValue, oldValue) {
-      var i = [];
-      var items = [];
-      var ignore = $scope.ignoreList;
-      var updateflag = 0;
-
-      if (newValue) {
-        for (i = 0; i < newValue.length; i++) {
-          if (ignore.indexOf(newValue[i].name) < 1) {
-            items.push(newValue[i]);
-          }
-        }
-
-        if ($scope.cloud) {
-          if (oldValue) {
-            if (oldValue.length !== newValue.length) {
-              updateflag = 1;
-            } else {
-              for (i = 0; i < newValue.length; i++) {
-                if (!updateflag & newValue[i].name !== oldValue[i].name & newValue[i].score !== oldValue[i].score) {
-                  updateflag = 1;
+    $scope.$watch('words', function (newValue, oldValue) {
+      angular.element(document).ready(function () {
```

Comment:
Please remove this document.ready, or convince me for the need for this with a jsfiddle or such..